hash/zx81_cass.xml

```diff
@@ -2253,13 +2330,27 @@ Known dumps not yet added (as of 2023-06-14):
 		<description>Flug-Simulation</description>
 		<year>1982</year>
 		<publisher>Sinclair Research</publisher>
+		<info name="usage" value="16K" />
 		<part name="cass1" interface="zx81_cass">
 			<dataarea name="cass" size="13855">
 				<rom name="FlightSimulation.tzx" size="13855" crc="945da3c1" sha1="de020d7531c0904a8a152193e7b3d5203e5c1017"/>
 			</dataarea>
 		</part>
 	</software>
 
+	<software name="flsimul_txp" cloneof="flsimul">
+		<description>Simulador de Voo (Timex Portugal)</description>
```

Comment:
Again, no need to translate the title for the shortname, something like 'simdevoo' is suitable.